scripts/simplify_network.py

```diff
@@ -975,14 +1000,14 @@ def merge_isolated_nodes(n, threshold, aggregation_strategies=dict()):
         "exclude_carriers", []
     )
     hvdc_as_lines = snakemake.params.electricity["hvdc_as_lines"]
-    aggregation_strategies = snakemake.params.cluster_options.get(
-        "aggregation_strategies", {}
-    )
-    # translate str entries of aggregation_strategies to pd.Series functions:
-    aggregation_strategies = {
-        p: {k: getattr(pd.Series, v) for k, v in aggregation_strategies[p].items()}
-        for p in aggregation_strategies.keys()
-    }
+    # aggregation_strategies = snakemake.params.cluster_options.get(
+    #    "aggregation_strategies", {}
+    # )
+    ## translate str entries of aggregation_strategies to pd.Series functions:
+    # aggregation_strategies = {
+    #    p: {k: getattr(pd.Series, v) for k, v in aggregation_strategies[p].items()}
+    #    for p in aggregation_strategies.keys()
+    # }
```

Comment:
Can you explain?
No more needed?

Reply:
Yes, the format transformation is not needed anymore. The aggregation strategies are [read directly](https://github.com/ekatef/pypsa-earth/blob/cefc09ce5e6afe2ad5b4a60aa3552dcc376dbcc6/scripts/cluster_network.py#L737) from the config via Snakemake parameters.